CauldronMods/Controller/Environments/SuperstormAkela/CardSubClasses/SuperstormAkelaCardController.cs

```diff
@@ -225,8 +225,8 @@ private IEnumerator RefreshUI(Card card)
             IEnumerator coroutine2;
 
             // grab the card source for some other card visible to this card so the action can finish after initial flip
-            Card otherCard = FindCardsWhere(c => c.IsInPlayAndHasGameText && c.Title != Card.Title && GameController.IsCardVisibleToCardSource(c, GetCardSource())).TakeRandomFirstOrDefault(Game.RNG);
-            
+            Card otherCard = FindCardsWhere(c => c.IsInPlayAndHasGameText && c.Title != card.Title && GameController.IsCardVisibleToCardSource(c, GetCardSource())).TakeRandomFirstOrDefault(Game.RNG);
```

Comment:
why is this random, instead of just taking the first one?

Reply:
I'm not sure why it was done that way. I didn't change it because I don't think the random aspect is causing any issues.

Reply:
I mean this is a clear improvement so definitely good to merge, but all the tests still pass if it's just `.First()`